### PR TITLE
Additional unit tests

### PR DESCRIPTION
**Pull request by PepperMoJ**

### CODE REVIEW

File: test/services/test_gandi_service.py [hunk 1]
```diff
@@ -13,20 +13,59 @@ def test_valid_certificate_list_returns_expected_domain(self):
...
+
+    def test_multiple_valid_certificates_returned(self):
+        test_case_count = 3
+        response = GandiService(test_config, 'api_key', 'base_url', 'url_ext') \
+            .get_certificates_in_valid_state(
+            TestData.generate_multiple_gandi_certificate_states(
+                'valid', test_case_count),
+            TestData.generate_multiple_email_list(test_case_count)
+        )
+        for i in range(test_case_count):
```
Comment (connormaglynn) [inline test/services/test_gandi_service.py:48]: Best avoiding using logic during the tests such as `for` loops. The reason is that the tests are there to test the logic, so if there's logic in the tests, then who's testing the tests! 🧪 

I believe the `assertIn()` should cover the functionality you need here, though - i.e. something like 
```python
self.assertIn([TestData.test_domain_name_root + "0", TestData.test_domain_name_root + "1", TestData.test_domain_name_root + "2"], response)
```

or if that doesn't work

```python
self.assertIn(TestData.test_domain_name_root + "0", response)
self.assertIn(TestData.test_domain_name_root + "1", response)
self.assertIn(TestData.test_domain_name_root + "2", response)
```

Doesn't look as clean, but a nicer standard to reduce the complexity when debugging tests 😄

File: test/resources/test_data.py [hunk 2]
```diff
@@ -2,24 +2,44 @@
 
 
 class TestData:
-    test_domain_name = "test.domain.gov.uk"
+    test_domain_name_root = "test.domain.gov.uk"
```
Comment (connormaglynn) [inline test/resources/test_data.py:5]: love the verbose names ❤️ 😁

**Review by connormaglynn — APPROVED** [review]
LGTM 🚀

File: test/services/test_gandi_service.py (hunk 1 above)
Comment (PepperMoJ) [reply test/services/test_gandi_service.py:48]: Cool beans, I'll move everything out to separate asserts 🙌

**Pull request closed without merging**